app/app.rb

```diff
@@ -44,7 +44,8 @@ class DwellBNB < Sinatra::Base
   post '/spaces' do
     space = Space.new(name: params[:name],
                 description: params[:description],
-                price: params[:price], picture_url: params[:picture_url])
+                price: params[:price], picture_url: params[:picture_url],
+                user_id: current_user.id)
```

Comment:
might need a flash error or HTML error or the spaces wont save if not logged in

Reply:
Sorted